website/docs/installation.md

```diff
@@ -27,12 +31,36 @@ The easiest way to install Docusaurus is to use the command line tool that helps
 npx create-docusaurus@latest [name] [template]
 ```
 
-Example:
+To create a new Docusaurus app, you may choose one of the following package manager that you like:
+
+<Tabs>
+<TabItem value="npm/npx">
 
 ```bash
+# Modern way available since npm v6+
+npm init docusaurus website classic
+
+# Outdated way available since npm v5.2+
 npx create-docusaurus@latest website classic
```

Comment:
`npx` is the only command commonly known to execute a package script from the NPM registry which doesn't permanently install it locally, so in #5797 `npm init`/`yarn create` was replaced with this. It was not meant to mean "npm", although we do know that npm provides this command. In Yarn berry there's `yarn dlx`, but it's not that popular.